### DISCUSSION TITLE
dimmer like sonoff t3?

### DISCUSSION THREAD

**Discussion by lukasa1993** [arendst/Tasmota#11211] [Q&A] [answered]
i really like sonoff t3 design is there any dimmer which look like this maybe same looks as 3gang ? 

![iu](https://user-images.githubusercontent.com/1499012/109823710-9c375800-7c51-11eb-9b2a-496170db8f01.jpeg)

**Comment by ascillato2** [comment, marked as answer]
Please, check tasmota.com

If there is something like that already supported, it will be listed in the templates.
